hydra-core/hydra-core.gemspec

```diff
@@ -16,10 +16,11 @@ Gem::Specification.new do |gem|
   gem.version       = version
   gem.license       = "APACHE2"
 
-  gem.required_ruby_version = '>= 1.9.3'
+  gem.required_ruby_version = '>= 2.4'
 
   gem.add_dependency 'hydra-access-controls', version
   gem.add_dependency "railties", '>= 4.0.0', '< 6'
+  gem.add_dependency 'sass-rails', '~> 5.0'
```

Comment:
As far as I know `hydra-core` doesn't really depend on `sass-rails`.  Is there another way that avoids adding it as a dependency?  Maybe a dev dependency if necessary?

Reply:
Removed.